ui/apps/dashboard/src/app/(organization-active)/(dashboard)/env/[environmentSlug]/functions/FunctionTable.tsx

```diff
@@ -97,10 +97,10 @@ export function FunctionTable({ rows = [] }: Props) {
   );
 }
 
-function Shimmer() {
+function Shimmer({ className }: { className?: string }) {
   return (
-    <div className="flex">
-      <Placeholder className="mx-4 my-4 h-2.5 w-full bg-slate-200" />
+    <div className={`flex ${className}`}>
```

Comment:
**nitpick (non-blocking)**: You may want to use the [`cn()` utility](https://github.com/inngest/inngest/blob/main/ui/packages/components/src/Skeleton/Skeleton.tsx) to merge class names so that it can handle Tailwind CSS class conflicts. I've been burned before by style conflicts.